client/components/Recordings.jsx

```diff
@@ -26,60 +34,115 @@ export default class Recordings extends React.Component {
+  }
+
+  saveAudioToProfile(e) {
+    e.preventDefault();
+    const formData = new FormData();
+    const userId = 1;
+    const title = this.state.title;
+    const recordingLength = this.state.duration;
+    formData.append('userId', userId);
+    formData.append('title', title);
+    formData.append('recordingLength', recordingLength);
+    formData.append('audio', this.file, this.file.name);
+    fetch('/api/recordings', {
+      method: 'POST',
+      body: formData
+    })
+      .then(response => {
+        response.json();
+        this.setState({ formInputs: false, recordings: [] });
+      })
+      .catch(err => console.error(err));
+  }
+
+  discardAudio(e) {
+    e.preventDefault();
+    this.setState({ recordings: [], title: '', formInputs: false });
   }
 
   deleteAudio(audioUrl) {
-    const recordings = this.state.videos.filter(audio => audio !== audioUrl);
+    const recordings = this.state.recordings.filter(audio => audio !== audioUrl);
     this.setState({ recordings });
   }
 
   render() {
-    const { recordings, recording } = this.state;
+    const { recordings, recording, formInputs } = this.state;
     const recordInner = (
             <div className="outer-record-shell row justify-center-all">
               <div className="inner-record-shell">
               </div>
             </div>
     );
-    const buttonClassName = 'col-100 outline record-button row justify-center-all';
-
+    const recordButtonClassName = 'col-100 outline record-button row justify-center-all';
     return (
-      <>
+      <form onSubmit={this.saveAudioToProfile} action="submit" ref={this.form}>
+          <div className="row justify-center-all">
+            {formInputs &&
+            <div className="padding-input">
+            <input onChange={this.titleName} type="text" className="title-input" required/>
+            </div>
+            }
+          </div>
         <div className="row justify-center-all padding-record">
-          {!recording && <button onClick={e => this.startRecording(e)} className={buttonClassName}>{recordInner}</button>}
-          {recording && <button onClick={e => this.stopRecording(e)} className={buttonClassName}>{recordInner}</button>}
+          {!recording && <button onClick={e => this.startRecording(e)} className={recordButtonClassName}>{recordInner}</button>}
+          {recording && <button onClick={e => this.stopRecording(e)} className={recordButtonClassName}>{recordInner}</button>}
         </div>
         <div className="row justify-center-all padding-record">
-          {recordings.map(audioURL => (
-            <div key={audioURL}>
-              <audio controls="controls">
+        {recordings.map(audioURL => (
+          <div key={audioURL}>
+              <audio controls="controls" ref={this.audio}>
                 <source src={audioURL} />
               </audio>
-            </div>
-          )
-          )}
+          </div>
+        )
+        )}
         </div>
```

Comment:
I'm a little bit confused by this part. You are passing the same `ref` for every `audioURL`. Are you trying to make one recording or more than one?